.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "7"
+  - "8"
```

Comment:
It would be actually to build with node_js 6 as vscode-jsonrpc. And the bug with node 8 should be fixed first there.